prettier.config.js

```diff
@@ -11,7 +11,12 @@ module.exports = {
             files: [
                 'src/content/**/*',
                 'src/tests/unit/tests/DetailsView/**/*',
-                'src/tests/unit/tests/injected/**/*',
+                'src/tests/unit/tests/injected/adapters/**/*',
+                'src/tests/unit/tests/injected/analyzers/**/*',
+                'src/tests/unit/tests/injected/components/**/*',
+                'src/tests/unit/tests/injected/frameCommunucators/**/*',
```

Comment:
Seems like a typo here:
```suggestion
                'src/tests/unit/tests/injected/frameCommunicators/**/*',
```

Some files from the frameCommunicators subfolder have also been changed as a result.

Reply:
Thanks for catching this. I fix the typo and ran `yarn format:fix` again.